examples/analytics/main.tf

```diff
@@ -51,7 +49,7 @@ resource "oci_analytics_analytics_instance" "test_oce_instance_public" {
aXRfciB1cm46b3BjOmlkbTp0LmpvYi5hcHAgdXJuOm9wYzppZG06dC5vYXV0aGNvbnNlbnRzIHVybjpvcGM6aWRtOnQudXNlcnNfciB1cm46b3BjOmlkbTp0LnNvbWkgdXJuOm9wYzppZG06Zy5zaGFyZWRmaWxlcyB1cm46b3BjOmlkbTp0LmhlbHBkZXNrLnVzZXIgdXJuOm9wYzppZG06dC5yZXMuaW1wb3J0ZXhwb3J0IHVybjpvcGM6aWRtOnQuam9iLmlkZW50aXR5IHVybjpvcGM6aWRtOnQuY3VzdG9tY2xhaW1zIHVybjpvcGM6aWRtOnQuc2FtbCB1cm46b3BjOmlkbTp0Lm1mYSB1cm46b3BjOmlkbTp0LmRiLmFkbWluIHVybjpvcGM6aWRtOnQuc2NoZW1hcyB1cm46b3BjOmlkbTp0Lm1mYS51c2VyYWRtaW4gdXJuOm9wYzppZG06dC51c2VyLm1hbmFnZXIuam9iIHVybjpvcGM6aWRtOnQub2F1dGggdXJuOm9wYzppZG06dC5ncm91cHMgdXJuOm9wYzppZG06dC5qb2IuaW1wb3J0ZXhwb3J0IHVybjpvcGM6aWRtOnQuaWRicmlkZ2UudW5tYXBwZWQuaWRjc2F0dHJpYnV0ZXMgdXJuOm9wYzppZG06dC5rcmIuYWRtaW4gdXJuOm9wYzppZG06dC5uYW1lZGFwcGFkbWluIHVybjpvcGM6aWRtOnQuYmxrcnB0cyB1cm46b3BjOmlkbTp0LnNlbGZyZWdpc3RyYXRpb25wcm9maWxlIHVybjpvcGM6aWRtOnQudXNlci5hdXRoZW50aWNhdGUgdXJuOm9wYzppZG06dC5ncmFudHMgdXJuOm9wYzppZG06dC5hdXRoZW50aWNhdGlvbiB1cm46b3BjOmlkbTp0LmNvbnRhaW5lciB1cm46b3BjOmlkbTp0LmltYWdlcyB1cm46b3BjOmlkbTp0LmNhLmFkbWluIHVybjpvcGM6aWRtOnQuYnVsayB1cm46b3BjOmlkbTp0LmRlbGVnYXRlZC5ncm91cC5tZW1iZXJzIHVybjpvcGM6aWRtOnQuam9iLnNlYXJjaCB1cm46b3BjOmlkbTp0LmlkYnJpZGdlIHVybjpvcGM6aWRtOnQuc2V0dGluZ3MgdXJuOm9wYzppZG06dC51c2VyLm1hbmFnZXIuc2VjdXJpdHkgdXJuOm9wYzppZG06dC5jbG91ZGdhdGUgdXJuOm9wYzppZG06dC5pZGJyaWRnZS5zb3VyY2VldmVudCB1cm46b3BjOmlkbTp0LnBvbGljeSB1cm46b3BjOmlkbTp0LnVzZXJzIHVybjpvcGM6aWRtOnQucmVwb3J0cyB1cm46b3BjOmlkbTpnLmlkY3NycHRzbWV0YV9yIiwiY2xpZW50X3RlbmFudG5hbWUiOiJpZGNzLTlmZWIzZWY5MWNhNDQ0MWJiZWRhZGJjNTY0MGQ1ZmI4IiwidXNlcl9sYW5nIjoiZW4iLCJ1c2VyQXBwUm9sZXMiOlsiQXV0aGVudGljYXRlZCIsIkdsb2JhbCBWaWV3ZXIiLCJJZGVudGl0eSBEb21haW4gQWRtaW5pc3RyYXRvciJdLCJleHAiOjE2MDA5OTIwMTgsImlhdCI6MTYwMDk4ODQxOCwiY2xpZW50X2d1aWQiOiJlMmM1Y2ZjZGY3ZWM0ZmVmOTNhYzRlZDIzZTk3OTBjZiIsImNsaWVudF9uYW1lIjoiVGVzdGluZ1NESyIsImlkcF90eXBlIjoiTE9DQUwiLCJ0ZW5hbnQiOiJpZGNzLTlmZWIzZWY5MWNhNDQ0MWJiZWRhZGJjNTY0MGQ1ZmI4IiwianRpIjoiYjAyNzJiYWItM2Y1MC00YTMzLWJhZjYtN2RmMjg4MzdiNDRkIiwidXNlcl9kaXNwbGF5bmFtZSI6IlBhcm5lZXQgS2F1ciIsImFsc29UZW5hbnRzIjpbImlkY3MtY2EiXSwic3ViX21hcHBpbmdhdHRyIjoidXNlck5hbWUiLCJwcmltVGVuYW50Ijp0cnVlLCJ0b2tfdHlwZSI6IkFUIiwiY2FfZ3VpZCI6ImNhY2N0LTA2Y2JlODcwODExMzQwY2E4ZGJlMWU4NTgwYWFkZmI2IiwiYXVkIjpbImh0dHBzOlwvXC9pZGNzLTlmZWIzZWY5MWNhNDQ0MWJiZWRhZGJjNTY0MGQ1ZmI4LmlkZW50aXR5Lm9yYWNsZWNsb3VkLmNvbTo0NDMiLCJ1cm46b3BjOmxiYWFzOmxvZ2ljYWxndWlkPWlkY3MtOWZlYjNlZjkxY2E0NDQxYmJlZGFkYmM1NjQwZDVmYjgiXSwidXNlcl9pZCI6IjJlNWIzZmE3MmI1YTQ2OTFiOGZiOTNkMDY0NzYwNmE3IiwiY2xpZW50QXBwUm9sZXMiOlsiQXV0aGVudGljYXRlZCBDbGllbnQiLCJJZGVudGl0eSBEb21haW4gQWRtaW5pc3RyYXRvciJdLCJ0ZW5hbnRfaXNzIjoiaHR0cHM6XC9cL2lkY3MtOWZlYjNlZjkxY2E0NDQxYmJlZGFkYmM1NjQwZDVmYjguaWRlbnRpdHkub3JhY2xlY2xvdWQuY29tOjQ0MyJ9.pS85ZctqAADIgBv2u8yjq8zZKjimsNyosontgJqUkSzx6H1Rd7mQqSRMb395_cJ7AHz4M7KRDNdOSZWmP9OwCFQzkYoeQNWGCJM0gKJxxWAvbqLp048Nl2Ch2MOq8oA3h2BfDxJ-NKH0iAeQrEMeh8G3Ki3g4fACbPbFsKDv5MdII4ey-0xxskkbU3zCfgN7iRvSPGDyb65zxbkGsYZan_NZg_yCdJPz7EME2p8o5jsNxjCANKxPnJQ9U88JFvkQq6RPTvsihgOI4CtBu12Faa71VuFWltza0dwtbI8z8QImTKYx84WQgWUtszQ_DKxYJvfCeEzAhsulUSeBvaE-Mg"
```

Comment:
we shouldn't expose idcs token data in example

Reply:
addressed